test/e2e/specs/end_user.go

```diff
@@ -42,6 +44,7 @@ var _ = Describe("Openshift on Azure end user e2e tests [EndUser]", func() {
 		namespace = "e2e-test-" + namespace
 		err = cli.CreateProject(namespace)
 		Expect(err).ToNot(HaveOccurred())
+		log = &logrus.Entry{}
```

Comment:
Should we just create this logger in the var section above?

Reply:
It would be more efficient to do this in the var section above. Currently a different logger is being created for every test spec.